ansari_db.py

```diff
@@ -49,15 +35,21 @@ def log(self, role, content, function_name=None):
 class AnsariDB:
     """Handles all database interactions."""
 
-    db_url = os.getenv("DATABASE_URL", "postgresql://mwk@localhost:5432/mwk")
-    token_secret_key = os.getenv("SECRET_KEY", "secret")
-    ALGORITHM = "HS256"
-    ENCODING = "utf-8"
```

Comment:
I don't think these should be settings. ALGORITHM and ENCODING change the format of DB keys. If you mess with these settings, it breaks the DB.

Reply:
I set them as constants.